pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -139,6 +139,19 @@ type TaskRunStatus struct {
 	TaskRunStatusFields `json:",inline"`
 }
 
+// TaskRunConditionType is an enum used to store TaskRun custom conditions
+// conditions such as one used in spire results verification
+type TaskRunConditionType string
+
+const (
+	// TaskRunConditionResultsVerified is a Condition Type that indicates that the results were verified by spire
+	TaskRunConditionResultsVerified TaskRunConditionType = "SignedResultsVerified"
+)
```

Comment:
If I understand correctly, this is a new condition that will exist in parallel to the `Succeeded` one, right?